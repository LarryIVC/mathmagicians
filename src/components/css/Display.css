.Display {
  text-align: right;
  background-color: #858593;
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: flex-end;
  padding: 20px;
  color: #fff;
}
