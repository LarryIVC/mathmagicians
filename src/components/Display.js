import React from 'react';
import './Display.css';

function Display() {
  return (
    <div className="Display">0</div>
  );
}

export default Display;
