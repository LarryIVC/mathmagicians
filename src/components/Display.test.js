import React from 'react';
import { render } from '@testing-library/react';
import Display from './Display';

test('renders Display component with input', () => {
  const input = {
    total: 10,
    operation: '+',
    next: 5,
  };
  const { container } = render(<Display input={input} />);
  expect(container).toMatchSnapshot();
});
