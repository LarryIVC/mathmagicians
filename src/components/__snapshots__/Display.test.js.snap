// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Display component with input 1`] = `
<div>
  <div
    class="Display"
  >
    10
    +
    5
  </div>
</div>
`;
